services/networking/loadbalancer/taxonomy.md

```diff
@@ -0,0 +1,67 @@
+| CCC-050320  | Full start                                            | Load balancer can be configured to allow targets to receive its full share of requests as soon as it's registered.                                                                                                                                                                                                            |
+| CCC-050321  | Slow start                                            | Load balancer can be configured to allow targets time to warm up before sending them a full share of requests.                                                                                                                                                                                                                |
+| CCC-050322  | Connection draining (deregistration delay)            | Ability to gracefully terminate active connections to backend instances before they are removed from the load balancer's pool, to allow ongoing requests to complete successfully without disruption.                                                                                                                         |
+| CCC-050323  | Flow termination / Proxy behavior                     | Proxy incoming requests to the selected backend server or servers, establishing a new connection to the backend server and forwarding the request while ensuring the integrity of the original request is maintained.                                                                                                         |
+| CCC-050324  | URL redirects                                         | Redirect incoming traffic to a different URL or location.                                                                                                                                                                                                                                                                     |
+| CCC-050325  | URL rewrites                                          | Rewrite URL paths before forwarding them to backend services.                                                                                                                                                                                                                                                                 |
+| CCC-050326  | Configurable timeouts                                 | Configure timeout settings related to the handling of connections and requests by the load balancer. These settings may include idle connection timeouts, request processing timeouts, and health check timeouts                                                                                                              |
```

Comment:
Should these be separated out into the different timeout configuration options?

Reply:
Leave it as is for now: those distinctions could be a later change, if required.